pkg/vulnsrc/vulnerability/vulnerability.go

```diff
@@ -210,7 +210,9 @@ func NormalizePkgName(ecosystem types.Ecosystem, pkgName string) string {
 		// and MUST consider hyphens and underscores to be equivalent.
 		pkgName = strings.ToLower(pkgName)
 		pkgName = strings.ReplaceAll(pkgName, "_", "-")
-	} else if ecosystem != NuGet { // Nuget is case-sensitive
+	}
+	// Nuget is case-sensitive, Go has case-sensitive packages
+	if ecosystem != NuGet && ecosystem != Go {
```

Comment:
Pip also gets into this if-statement. It should be `else if`.

Reply:
oh, sure. my mistake.